schemas/data/measure.schema.json

```diff
@@ -20,6 +20,12 @@
           "description":
             "Links to the `Metric` that this measure refers to. The `@type` can be omitted in contexts where it is obvious what metric is being referred to, for instance in the `Metrics` object that is keyed with `Metric` URIs."
         },
+        "@id": {
+          "title": "Unique Identifier",
+          "type": "string",
+          "format": "uri",
+          "description": "Unique identifier of the metric."
```

Comment:
Shouldn't this be: "Unique identifier of the *measure*? We have the metric identified using `@type` above.

If yes: then let's also turn this into a `xdm:id` or `xdm:uniqueMeasure`, make it a plain `string` and use a description along the lines of: 
> In cases of data collection using lossy communication channels, such as mobile apps or websites with offline functionality, where transmission of measures cannot be ensured, this property contains a client-generated, unique ID of the measure taken. 
> It is best practice to make this sufficiently long to ensure enough entropy. Additionally, if information such as time stamp, device ID, IP or MAC address, or other potentially user-identifying values are incorporated in the generation of the `xdm:uniqueMeasure`, the result should be hashed, so that no PII is encoded in the value, as the goal is not to identify user or device, but the specific measure in time.

It is probably also a good idea to set a minimum length of 20 characters.

If not: then just close the PR because it's a duplicate.

Reply:
@trieloff updated.